packages/api/cms-api/src/kubernetes/kubernetes.service.ts

```diff
@@ -39,6 +39,10 @@ export class KubernetesService {
         }*/
     }
 
+    getHelmRelase(): string {
```

Comment:
Why not use `get helmRelease(): string` instead? This would hide the function call in the builds resolver

Reply:
I think `this.kubernetesService.helmRelease` is a bit odd, but I've done it as requested.